test/to_vec.jl

```diff
@@ -93,6 +93,16 @@ end
             )
         end
 
+        @testset "Factorizations" begin
+            for dims in [(5, 5), (4, 6), (7, 3)]
+                M = randn(T, dims...)
+                P = M * M' + I  # Positive definite matrix
+                test_to_vec(svd(M); check_inferred = false)
```

Comment:
Is it an issue that type inference fails here?

Reply:
I fixed the inferrence.